src/data_provider/src/packages/packageLinuxDataRetriever.h

```diff
@@ -88,10 +87,11 @@ class FactoryPackagesCreator<LinuxType::STANDARD> final
                 getRpmInfo(callback);
             }
 
-            if (Utils::existsDir(APK_PATH))
+            if (Utils::existsDir(SNAP_PATH))
```

Comment:
I think we shouldn't replace the `APK_PATH` implementation.

Reply:
Disregard this change made because the l function was replaced
![image](https://user-images.githubusercontent.com/93778492/233643436-87b7d4b9-0970-4d24-a87f-ced9dad8d1d6.png)

Reply:
I understood that this was no longer necessary, maybe we should review this change.

Reply:
Why do you think this is no longer necessary? this is present in the master branch and it is part of the Alpine Linux package inventory support implementation.